README.md

```diff
@@ -32,6 +32,8 @@ If you select `ETHERNET` with `UBLOX_ODIN_EVK_W2` you must add this to your `tar
             }
 ```
 
+If you select `WNC14A2A`, you must include the `BufferedSerial` library (mbed add http://os.mbed.com/users/sam_grove/code/BufferedSerial/) because the WNC14A2A communicates via a UART interface.
```

Comment:
Let me know, but I ran into this even with the latest mbed-os release (5.7)

Reply:
Mbed OS has the UartSerial -class, which provides similar functionality to BufferedSerial. See https://github.com/ARMmbed/mbed-os/blob/master/drivers/UARTSerial.h

Reply:
It was moved into mbed OS, but renamed to avoid a name collision, but potentially increase confusion. It now exists as `UARTSerial`. It's had a few changes and improvements - but any code using `BufferedSerial` should be able to adapt easily.

Looking at your existing code I see you're suffering from the fact that ` BufferedSerial` was totally disjoint from the rest of the I/O system. `UARTSerial` solves that, so you can now do:

    UARTSerial serial(RX,TX);
    FILE *out = fdopen(&serial, "w");
    fprintf(out, "Hello!\n");